UI/src/app/models/arrow.ts

```diff
@@ -110,33 +110,52 @@ export class Arrow implements IConnector {
     );
 
     this.renderer.setAttribute(this.path, 'startXY', `${x1},${y1}`);
-    this.renderer.setAttribute(this.path, 'endXY', `${x1},${y1}`);
+    this.renderer.setAttribute(this.path, 'endXY', `${x2},${y2}`);
   }
 
   attachButton(button) {
     this.button = button;
   }
 
+  setEndMarkerType(type: string): void {
+    this.refreshPathHtmlElement();
+
+    const isActive = this.svgPath.attributes[9].value.includes('active');
```

Comment:
9 is magic number there. It's not obvious why we use it

Reply:
This is the index of the item in attributes. I can move it to constant. Will it be enough?

Reply:
moved